index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>TODO List</title>
+  </head>
+  <body>
+    <h1>ToDo List</h1>
+
+    <div class="btn">
+      <div>
+        <input class="input-item"></input>
+      </div>
+      <div>
+        <button class="add-btn">Add</button>
+      </div>
+      <button class="reset-btn">Reset</button>
+    </div>
+    <script src="script.js"></script>
```

Comment:
Script tag can be added in the head tag as well, why cant we add it there? What difference will it make

Reply:
Html runs in synchronous manner , if we add script tag in header than first script tag get excited at that time our button or any input filed is not initialised. So if we want to add it in header we have to add a async Attribute with it.